x/ibc/02-client/types/codec.go

```diff
@@ -52,19 +64,13 @@ func PackClientState(clientState exported.ClientState) (*codectypes.Any, error)
 	return anyClientState, nil
 }
 
-// MustPackClientState calls PackClientState and panics on error.
-func MustPackClientState(clientState exported.ClientState) *codectypes.Any {
-	anyClientState, err := PackClientState(clientState)
-	if err != nil {
-		panic(err)
-	}
-
-	return anyClientState
-}
-
 // UnpackClientState unpacks an Any into a ClientState. It returns an error if the
 // client state can't be unpacked into a ClientState.
 func UnpackClientState(any *codectypes.Any) (exported.ClientState, error) {
+	if any == nil {
+		return nil, errors.New("protobuf Any message cannot be nil")
```

Comment:
we should register an error if we use these in the message handlers to unpack stuff otherwise it'll be an internal error

Reply:
I realize we should probably change this everywhere. Originally it was fine when we just used this for genesis, but now we use these funcs everywhere

Reply:
changed the ones from this file only